src/test/a11y/a11y.ts

```diff
@@ -26,21 +27,13 @@ beforeAll((done /* call it or remove it*/) => {
 });
 
 function ensurePageCallWillSucceed(url: string): Promise<void> {
```

Comment:
This function only has one line and it is just calling axios to go to the URL. The function name is "ensurePageCallWillSucceed" and does not reflect what it is doing. I think it is clearer to remove this function and call axios.get(url) directly from the function testAccessibilityWithActions.